### PR TITLE
9.2: Switch to using openvswitch from 9.0

### PR DESCRIPTION
**Pull request by cgwalters**
The COPR thing breaks our lockfile handling, which wants to find the same NEVRAs across architectures.

Further, trying to ship a COPR repo just gives me the willies; there's absolutely no required reviews for changes or CI etc.

If we need fixes post-9.0 for openvswitch, then we'll do something custom.

### CODE REVIEW

File: manifest-rhel-9.2.yaml [hunk 1]
```diff
@@ -150,3 +142,6 @@ repo-packages:
       - cri-tools
       - openshift-clients
       - openshift-hyperkube
+  - repo: rhel-9.0-fast-datapath
```
Comment (jlebon) [inline manifest-rhel-9.2.yaml:145]: Aside: I've been thinking for a while we should support specifying repos here that aren't listed in the toplevel `repos` key. The semantic would be that we enable this repo only for the specified packages and nothing else.

**Review by jlebon — APPROVED** [review]
/approve
/lgtm

**Comment by cgwalters** [comment]
> error: Unknown rpm-md repository: rhel-9.0-fast-datapath

Hmm digging

**Comment by cgwalters** [comment]
/override ci/prow/rhcos-86-build-test-metal
was a flake but anyways doesn't matter since rhel8 will be gone after this PR

**Pull request merged**